src/targets/chrome/get-selector-box-size.js

```diff
@@ -1,10 +1,42 @@
 const getSelectorBoxSize = (window, selector) => {
-  const element = window.document.querySelector(selector);
-  if (!element) {
-    throw new Error('Unable to find element');
+  const elements = [...window.document.querySelectorAll(selector)];
```

Comment:
Prefer `Array.from` here

Reply:
This one is faster. 🤓 

https://jsperf.com/array-from-vs-spread-syntax

<img width="770" alt="screen shot 2018-07-06 at 15 59 40" src="https://user-images.githubusercontent.com/3720424/42382805-ab8be858-8135-11e8-9d15-71140bb20caa.png">

<img width="768" alt="screen shot 2018-07-06 at 15 59 30" src="https://user-images.githubusercontent.com/3720424/42382807-adfb985e-8135-11e8-8221-d755708ea7c2.png">

<img width="780" alt="screen shot 2018-07-06 at 15 59 10" src="https://user-images.githubusercontent.com/3720424/42382809-afc41738-8135-11e8-8446-b96a09e8b542.png">

Reply:
Mostly that `Array.from` has wider browser support, but given that we currently only support chrome anyways you can keep it 👍